Header header
float32 ax_c1
float32 ay_c1
float32 az_c1
float32 aex_c1
float32 aey_c1
float32 aez_c1

float32 ax_c2
float32 ay_c2
float32 az_c2
float32 aex_c2
float32 aey_c2
float32 aez_c2

float32 ax_c3
float32 ay_c3
float32 az_c3
float32 aex_c3
float32 aey_c3
float32 aez_c3

float32 ax_c4
float32 ay_c4
float32 az_c4
float32 aex_c4
float32 aey_c4
float32 aez_c4

float32 ax_c5
float32 ay_c5
float32 az_c5
float32 aex_c5
float32 aey_c5
float32 aez_c5

float32 ax_c6
float32 ay_c6
float32 az_c6
float32 aex_c6
float32 aey_c6
float32 aez_c6

float32 ax_c7
float32 ay_c7
float32 az_c7
float32 aex_c7
float32 aey_c7
float32 aez_c7

float32 ax_c8
float32 ay_c8
float32 az_c8
float32 aex_c8
float32 aey_c8
float32 aez_c8

